plugins/tracker-resources/src/components/projects/ProjectPresenter.svelte

```diff
@@ -0,0 +1,85 @@
+<!--
+// Copyright © 2022 Hardcore Engineering Inc.
+// 
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// 
+// See the License for the specific language governing permissions and
+// limitations under the License.
+-->
+
+<script lang="ts">
+  import { Ref } from '@anticrm/core'
+  import { Avatar, getClient } from '@anticrm/presentation'
+  import { Icon } from '@anticrm/ui'
+  import contact from '@anticrm/contact'
+  import { Project, ProjectStatus, Team } from '@anticrm/tracker'
+  import plugin from '../../plugin'
+  import ProjectStatusSelector from './ProjectStatusSelector.svelte'
+
+  export let value: Project
+  export let space: Ref<Team>
+
+  const client = getClient()
+
+  async function getLeadAvatar() {
+    if (!value.lead) {
+      return null
```

Comment:
Please use $lookup with already obtained value of lead, instead of individual requests, since you use projects with your table presenter only for now.

Reply:
Done